spec/controllers/styled_forecasts_controller_spec.rb

```diff
@@ -63,4 +101,75 @@
       expect(response).to render_template("show")
     end
   end
+
+  describe "GET :update" do
+    let(:forecasts) do
+      ForecastFactory.build(forecast_from_api)
+    end
+
+    let(:tag_builder) do
+      instance_double(Turbo::Streams::TagBuilder, replace: true)
+    end
+
+    before do
+      allow(Turbo::Streams::TagBuilder).to receive(:new).and_return(tag_builder)
+    end
+
+    context "when a recognised _day_ parameter is received" do
+      describe "when the day is _today_" do
+        it "passes the first forecast to the view" do
+          allow(CercApiClient).to receive(:forecasts_for).and_return(forecasts)
+
+          get :update, params: {day: :today}
+
+          expect(CercApiClient).to have_received(:forecasts_for).with("Southwark")
+          expect(tag_builder).to have_received(:replace).with(
+            "day_predictions",
+            partial: "predictions",
+            locals: {forecast: forecasts.first}
+          )
+        end
+      end
+
+      describe "when the day is _tomorrow_" do
+        it "passes the first forecast to the view" do
```

Comment:
I think this should be `it "passes the *second* forecast to the view"`!

and similarly the day after tomorrow is the third forecast?